protocol-designer/src/assets/localization/en/shared.json

```
@@ -57,27 +60,43 @@
       "body6": "Please note that in order to run the updated protocol on your robot successfully, the OT-2 App and robot are required to be updated to version 3.10.0 or higher."
     }
   },
+  "message_exact_labware_match": "This labware is identical to one you have already uploaded.",
+  "message_invalid_json_file": "Protocol Designer only accepts custom JSON labware definitions made with our Labware Creator",
```

nit: modifier order
```suggestion
  "message_invalid_json_file": "Protocol Designer only accepts JSON custom labware definitions made with our Labware Creator",
```

Also this is kinda not true…whomst among us hasn't hacked a labware definition?

---

lol this is legacy text from 5 years ago, people totally don't always use Labware Creator

---

It's still good to give a gentle push in that direction 😇